package.json

```diff
@@ -36,7 +36,11 @@
         "url": "https://github.com/AdvisorySG/ghost-advisory-theme.git"
     },
     "bugs": "https://github.com/AdvisorySG/ghost-advisory-theme/issues",
-    "contributors": "https://github.com/AdvisorySG/ghost-advisory-theme/graphs/contributors",
+    "contributors": [{
+        "name" : "Advisory SG Ghost Team",
+        "email" : "contact@advisory.sg",
```

Comment:
...and this to "technology@advisory.sg"

Reply:
https://github.com/AdvisorySG/ghost-advisory-theme/blob/9955faf3f65711e2b1e7cd5c5636a2729c927720/package.json#L39-L43

Changes made